jaeger/cmd/install.go

```diff
@@ -68,27 +67,28 @@ func newCmdInstall() *cobra.Command {
 		`Skip checks for namespace existence`,
 	)
 
-	// TODO: Add --set flag set and also config
+	flags.AddValueOptionsFlags(cmd.Flags(), &options)
 
 	return cmd
 }
 
-func install(w io.Writer, values *jaeger.Values) error {
+func install(w io.Writer, options values.Options) error {
+	// Merge and create final set of values
```

Comment:
I don't know if this comment is right.  Doesn't this just get the "overrides" yaml from the flags?  In other words, this doesn't include the default values from the chart at all.

Reply:
For clarity, I'd consider naming this `valuesOverrides` or something along those lines.